src/ensembl/stories/shared-components/eye-toggle-icon/EyeToggleIcon.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+import { storiesOf } from '@storybook/react';
+
+import EyeToggleIcon from 'src/shared/eye-toggle-icon/EyeToggleIcon';
+
+type Props = {
+  eyeStatus: string;
+};
+
+const EyeToggleIconParent = (props: Props) => {
+  const [eyeStatus, setVisible] = useState('on');
+
+  const toggleEye = () => {
+    if (eyeStatus === 'on') {
+      setVisible('off');
+      return;
+    }
+    setVisible('on');
+  };
+
+  const styles = {
+    margin: '10px'
+  };
+
+  return (
+    <div style={styles}>
+      <EyeToggleIcon
+        iconStatus={props.eyeStatus}
+        description={'enable/disable'}
+        callbackFunction={toggleEye}
+      />
+    </div>
+  );
+};
+
+storiesOf('Components|Shared Components/EyeToggleIcon', module)
+  .add('On', () => {
+    return <EyeToggleIconParent eyeStatus={'on'} />;
+  })
```

Comment:
Would just one story with a clickable icon/button be more representative of the button's behavior?

Reply:
Yes, that's a good point. I've updated it now.